fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/Operator.java

```diff
@@ -181,10 +185,26 @@ public RowOutputInfo getRowOutputInfo(List<OptExpression> inputs) {
         return rowOutputInfo;
     }
 
+    public ValueProperty getValueProperty(List<OptExpression> inputs) {
```

Comment:
why not implement the interface by visit mode?

Reply:
I think we may need get this property in other rules.  For example, we may want replcae column to constant